spec/rubocop/cop/style/pattern_argument_spec.rb

```diff
@@ -0,0 +1,263 @@
+# frozen_string_literal: true
+
+RSpec.describe RuboCop::Cop::Style::PatternArgument, :config do
+  context 'when using Ruby 2.5 or newer', :ruby25 do
+    %w[all? any? none? one?].map { |method| [method, '^' * method.length] }.each do |method, arrows|
```

Comment:
No need for arrows... Use `^{method}` and pass `method: method` to expect_offense 😄